kolibri/plugins/learn/assets/src/vue/toolbar/index.vue

```diff
@@ -0,0 +1,238 @@
+<template>
+
+  <div v-bind:class="['toolbar-show', displayToolBar ? 'toolbar-hide' : '' ]" >
+    <breadcrumbs class="breadcrumbs"></breadcrumbs>
+    <div :class="{ 'toggle-menu-on' : more }">
+      <label for="chan-select" :class="[ more ? 'lable-on' : 'visuallyhidden' ]" >Switch Channels</label>
+      <select
+        class="chan-select"
+        :class="[ more ? 'chan-select-mobile-location' : 'chan-select-location' ]"
+        id="chan-select"
+        name="chan-select"
+        v-model="getCurrentChannel"
```

Comment:
@indirectlylit Not quite the source. You can view what the issues were here: https://github.com/learningequality/kolibri/pull/411